packages/docs/src/helpers/styles.ts

```diff
@@ -35,7 +35,7 @@ export const renderStyles = (
 
   return html`
     <${usedTag} class=${classes}>
-      ${defaultSlot}
+      ${unsafeStatic(defaultSlot)}
```

Comment:
I guess this is needed for the table to work as a content element?

Reply:
I needed it for the table story in `docs/stories/styles/table.stories.ts`
to do this:

```js
args: overrideArgs([
    {
      name: 'default',
      type: 'slot',
      value: `<tbody>
    <tr>
      <td>
        Cell content
      </td>
    </tr>
  </tbody>`,
    },
  ], defaultArgs),
  ```

So if someone looks at the code of the story he does not get this:

```html
<table>Cell content</table>
```

but this: 
```html
<table>
  <tbody>
    <tr>
      <td>Cell content</td>
    </tr>
  </tbody>
</table>
```